cognite/client/_api/iam.py

```diff
@@ -100,8 +99,10 @@ def compare_capabilities(
             project (str | None): If a ProjectCapability or ProjectCapabilityList is passed, we need to know which CDF project
                 to pull capabilities from (existing might be from several). If project is not passed, and ProjectCapabilityList
                 is used, it will be inferred from the CogniteClient used to call retrieve it via token/inspect.
-            ignore_allscope_meaning (bool): Option on how to treat allScopes. When True, this function will return
-                e.g. an Acl scoped to a dataset even if the user have the same Acl scoped to all. Defaults to False.
+            ignore_allscope_meaning (bool): Option on how to treat scopes that encompass other scopes, like allScope. When True,
```

Comment:
When would you want this set to True?

Reply:
That is the most logical to me. You have access to all data sets when you have all, which would be the most common use case.

Reply:
If you want two groups of capabilities to be equal, and they differ in scopes "not all", then this option will tell you exactly what those differences are. For Peter's bootstrap CLI (and maybe others...) it means the manual work of removing any possible all-scopes or raw-database-scopes before comparing is not needed.